UnitTests/Views/TextValidateFieldTests.cs

```diff
@@ -19,14 +17,13 @@ public void Initialized_With_Cursor_On_First_Editable_Character ()
 			Width = 20
 		};
 
-		field.NewKeyDownEvent (new Key (KeyCode.D1));
+		field.NewKeyDownEvent (new (KeyCode.D1));
 
 		Assert.Equal ("--(1___)--", field.Provider.DisplayText);
 		Assert.Equal ("--(1   )--", field.Text);
 	}
 
 	[Fact]
-	[AutoInitShutdown]
```

Comment:
🥳

Reply:
That was already removed and only need to be merges with the v2_develop branch.

https://github.com/gui-cs/Terminal.Gui/blob/a3627aed5e4038b7b105cb98401472f3a5ed4fae/UnitTests/Views/TextValidateFieldTests.cs#L26-L27